app/_hub/kong-inc/app-dynamics/overview/_index.md

```diff
@@ -124,6 +124,11 @@ This location can be changed by setting the `KONG_APPD_LOGGING_LOG_DIR` environm
 If problems occur with the AppDynamics integration, inspect the AppDynamics agent's log files in addition to the Kong
 Gateway logs.
 
+{% if_version gte:3.10.x %}
+{:.important}
+> **Important:** ARM isn't supported when using the AppDynamics agent. The agent only supports x86 architecture.
```

Comment:
Can you double check that this is 3.10 only? I'm guessing this is actually version-agnostic.

Reply:
Sure, I'll check. The Jira ticket said "fix version: 3.10" so I went off that originally